boards/arm/mimx8mp_phyboard_pollux/mimx8mp_phyboard_pollux-pinctrl.dtsi

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2022 PHYTEC Messtechnik GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <nxp/nxp_imx/mimx8ml8dvnlz-pinctrl.dtsi>
+
+&pinctrl {
+	uart1_default: uart1_default {
+		group0 {
+			pinmux = <&iomuxc_uart1_rxd_uart_rx_uart1_rx>,
+				<&iomuxc_uart1_txd_uart_tx_uart1_tx>;
+			slew-rate = "slow";
+			drive-strength = "x1";
+		};
+	};
```

Comment:
I thought it makes sense to have every uart that is possible to use with the M7-Core preconfigured but set on disabled to make it easy to use it in an application. Maybe someone wants to use the uart1 for the M7-Core and another one for the Coretex-A53. Like this it can easily changed, because its preconfigured. Do you think different about that?